requester/requester.go

```diff
@@ -29,9 +29,39 @@ import (
 	"sync"
 	"time"
 
+	"github.com/vkuznet/x509proxy"
 	"golang.org/x/net/http2"
 )
 
+// Return client X509 certificates
+func Certs() (tls_certs []tls.Certificate) {
+	uproxy := os.Getenv("X509_USER_PROXY")
+	uckey := os.Getenv("X509_USER_KEY")
+	ucert := os.Getenv("X509_USER_CERT")
+	if len(uproxy) > 0 {
+		// use local implementation of LoadX409KeyPair instead of tls one
+		x509cert, err := x509proxy.LoadX509Proxy(uproxy)
+		if err != nil {
```

Comment:
return fmt.Errorf("failed to parse proxy X509 proxy set by X509_USER_PROXY: %v, err)